### PR TITLE
repair JetBrains WebStorm `DuplicateOptionError`

### PR DESCRIPTION
**Pull request by LucasLarson**
error generated:
```
    raise DuplicateOptionError(sectname, optname,
configparser.DuplicateOptionError: While reading from '/usr/local/Cellar/mackup/HEAD-c03c2a2/libexec/lib/python3.8/site-packages/mackup/applications/webstorm.cfg' [line 38]: option 'Library/Application Support/WebStorm2019.3' in section 'configuration_files' already exists
```
 

keep one pair of the two duplicated lines that break functionality:
https://github.com/lra/mackup/blob/c03c2a2efa54ee964216d4811c8143182a0329ec/mackup/applications/webstorm.cfg#L36-L39

 
error’s first appearance:
    [https://github.com/lra/mackup/commit/901eab6](https://github.com/lra/mackup/commit/901eab685ed44fe3da42ad0a49ada2ba562244a1#diff-4eda02476608891c73a4e51f70f17903R36-R39)

### CODE REVIEW

**Comment by rlam3** [comment]
@Lucas

> error generated:
> 
> ```
>     raise DuplicateOptionError(sectname, optname,
> configparser.DuplicateOptionError: While reading from '/usr/local/Cellar/mackup/HEAD-c03c2a2/libexec/lib/python3.8/site-packages/mackup/applications/webstorm.cfg' [line 38]: option 'Library/Application Support/WebStorm2019.3' in section 'configuration_files' already exists
> ```
> 
>  
> 
> keep one pair of the two duplicated lines that break functionality:
> 
> https://github.com/lra/mackup/blob/c03c2a2efa54ee964216d4811c8143182a0329ec/mackup/applications/webstorm.cfg#L36-L39
> 
>  
> error’s first appearance:
> [https://github.com/lra/mackup/commit/901eab6](https://github.com/lra/mackup/commit/901eab685ed44fe3da42ad0a49ada2ba562244a1#diff-4eda02476608891c73a4e51f70f17903R36-R39)

Were you able to find a workaround to this problem?

**Comment by rlam3** [comment]
> error generated:
> 
> ```
>     raise DuplicateOptionError(sectname, optname,
> configparser.DuplicateOptionError: While reading from '/usr/local/Cellar/mackup/HEAD-c03c2a2/libexec/lib/python3.8/site-packages/mackup/applications/webstorm.cfg' [line 38]: option 'Library/Application Support/WebStorm2019.3' in section 'configuration_files' already exists
> ```
> 
>  
> 
> keep one pair of the two duplicated lines that break functionality:
> 
> https://github.com/lra/mackup/blob/c03c2a2efa54ee964216d4811c8143182a0329ec/mackup/applications/webstorm.cfg#L36-L39
> 
>  
> error’s first appearance:
> [https://github.com/lra/mackup/commit/901eab6](https://github.com/lra/mackup/commit/901eab685ed44fe3da42ad0a49ada2ba562244a1#diff-4eda02476608891c73a4e51f70f17903R36-R39)


Were you able to find a workaround to this problem?

**Comment by LucasLarson** [comment]
> @lucas
> 
> > error generated:
> > ```
> >     raise DuplicateOptionError(sectname, optname,
> > configparser.DuplicateOptionError: While reading from '/usr/local/Cellar/mackup/HEAD-c03c2a2/libexec/lib/python3.8/site-packages/mackup/applications/webstorm.cfg' [line 38]: option 'Library/Application Support/WebStorm2019.3' in section 'configuration_files' already exists
> > ```
> > 
> > 
> > keep one pair of the two duplicated lines that break functionality:
> > https://github.com/lra/mackup/blob/c03c2a2efa54ee964216d4811c8143182a0329ec/mackup/applications/webstorm.cfg#L36-L39
> > 
> > error’s first appearance:
> > [https://github.com/lra/mackup/commit/901eab6](https://github.com/lra/mackup/commit/901eab685ed44fe3da42ad0a49ada2ba562244a1#diff-4eda02476608891c73a4e51f70f17903R36-R39)
> 
> Were you able to find a workaround to this problem?

@rlam3 Yes, this worked for me: https://github.com/lra/mackup/pull/1628/files

**Review by rlam3 — APPROVED** [review]

**Comment by rlam3** [comment]
@LucasLarson Thanks! Worked like a charm. Why is this issue showing up now? I'm surprised nobody caught this earlier...

**Comment by LucasLarson** [comment]
> @LucasLarson Thanks! Worked like a charm. Why is this issue showing up now? I'm surprised nobody caught this earlier...

I made the PR a while ago now. No idea, but I’m super surprised it’s still a problem no one else has complained about.

**Comment by rlam3** [comment]
@LucasLarson Yes, I am also wondering how often this package is merged... It seems like it is taking a while for the latest updates to show up for homebrew as well. 

I had to switch over to using --HEAD but found no documentation on this so added to the documentation
Hopefully, the maintainers of this repo will merge changes soon.

https://github.com/lra/mackup/pull/1646/files

**Comment by LucasLarson** [comment]
> I had to switch over to using --HEAD

I’m assuming you were using Homebrew for that&nbsp;– do you happen to know how to get `‑‑HEAD` using pip (from Ubuntu or Alpine Linux)?

**Comment by rlam3** [comment]
@LucasLarson I don't know how to use --HEAD with pip. Is there a specific way we are supposed to use mackup? Yes I am using homebrew and only found out how to update using homebrew with --HEAD to get the latest updates directly from github instead.

Please let me know if I need to provide more info.

**Comment by LucasLarson** [comment]
@rlam3: I just saw that `brew install mackup --HEAD` doesn’t update mackup to [its latest commit](/../lra/mackup/commit/HEAD), even when you first perform 
`brew doctor && brew update && brew upgrade`.

I performed `brew install mackup --HEAD` again and I’m now at [e71c129 insteada c03c2a2](c03c2a2..e71c129), but this problem’s still here
https://github.com/lra/mackup/blob/c03c2a2efa54ee964216d4811c8143182a0329ec/mackup/applications/webstorm.cfg#L36-L39

---
@lra: it’d be super awesome if this software had more frequent releases, but I know that can create more frequent headaches. Alternatively, #1628 is a quick non-controversial fix to duplicated code. Let me know if you’d like to see changes before it’s merged.

Keep up the good work&nbsp;&ndash; and thank you!

**Comment by rlam3** [comment]
https://github.com/lra/mackup/pull/1622 looks like it already was merged in Aug of this year. Latest pull is e71c129 and dates back to Oct as the latest master release. So I think it should technically be up to date unless the owner of the repo pulled out the update in another commit.... @LucasLarson

**Comment by lra** [comment]
Thanks for this. Looks like it only impacted webstorm users, that's why there wasn't too much noise around it.
I'll push a hotfix release soon to include it.

**Comment by lra** [comment]
Fix included in https://github.com/lra/mackup/releases/tag/0.8.31

**Pull request closed without merging**
